.github/workflows/vast.yaml

```diff
@@ -278,13 +260,14 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Delete existing CHANGELOG.md
-        run:
-          rm CHANGELOG.md
-      - name: Download new CHANGELOG.md
+      - name: Delete existing changelog
+        run: |
+          rm: web/src/pages/changelog.md
```

Comment:
Is this step still necessary? I would imagine it suffices to download (= overwrite) the existing one.

Reply:
The download doesn't overwrite but rather error if the file exists. We do want to have the stub file, though, otherwise local builds do not work.

Reply:
Okay, if the download fails otherwise, there's no way around it.